src/components/linksComponents/sign-in/signInModal.tsx

```diff
@@ -0,0 +1,99 @@
+import { FaTimes } from "react-icons/fa";
+import InputText from "@/elements/inputText/inputText";
+import { useState } from "react";
+import { toast, ToastContainer } from "react-toastify";
+import { API_SIGN_IN } from "../../../constants/api";
+
+import styles from "./signInModal.module.scss";
+import Modal from "../../../modal/modal";
+import { getApiResourse } from "../../../utils/network";
+import "react-toastify/dist/ReactToastify.css";
+
+interface MyState {
+  isOpen: boolean;
+  onClose: () => void;
+  isSignIn: () => void;
+}
+
+const SignInModal = ({ isOpen, onClose, isSignIn }: MyState): JSX.Element | null => {
+  toast.error("something wrong");
+  toast.success("well");
+  console.log("signin");
+  const [login, setlogin] = useState("");
+  const [userPassword, setUserPassword] = useState("");
+  const [error, setError] = useState(false);
+  const updateLogin = (value: string) => {
+    setlogin(value);
+  };
+  const updatePassword = (value: string) => {
+    setUserPassword(value);
+  };
+
+  const signIn = async (param: string, data = {}) => {
+    const res = await getApiResourse(param, data);
+    console.log(res);
+
+    if (res[0]) {
+      isSignIn();
+    }
+    if (!res[0]) {
+      setError(true);
+    }
+  };
+
+  const submitUser = () => {
+    signIn(API_SIGN_IN, {
+      email: login,
+      password: userPassword,
+    });
+  };
+
+  // useEffect(() => {
+  //   getResponse("http://localhost:3000/register", {
+  //     email: "nils123on@email.com",
+  //     password: "nil123",
+  //   });
+  // }, []);
+
+  if (error) {
+    return <ToastContainer draggable={false} />; // problems with ToastContainer, I believe it should be implemented in network.tsx,
```

Comment:
you should define toastContainer in the main component, and just show it via the toast() function. https://www.npmjs.com/package/react-toastify Please, check the docs.